view/customer_list.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+//Include Menu
+require('menu.php');
+//Inlcude Database class
+include '../functions/datalayer/database.class.php';
+//Include Customer Controller Class
+include '../functions/datalayer/CustomerDB.php';
+//Customer Entity Class
+include '../functions/models/entCustomer.php';
+
+//Create connecting with CustomerDB Class
+$CustomerDB = new CustomerDB();
+
+?>
+
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <!-- Page Container -->
+    <div class="page__content">
+        <!-- Bootstrap Container -->
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-sm-12">
+                    <br>
+                    <h1 class="customer__header">Overview Customers</h1>
+                    <br>
+                </div>
+            </div>
+            <!-- Table setting -->
+            <div class="row">
+                <div class="col-sm-12"></div>
+                <table class="table table-hover customer__table">
```

Comment:
Met de boostrap 'container' in plaats van 'container-fluid' class krijg je hetzeflde effect als de 50% width in je stylesheet.